vmdb/app/controllers/mixins/containers_common_mixin.rb

```diff
@@ -38,6 +38,19 @@ def show_container(record, controller_name, display_name)
         pluralize(@view.extras[:total_count] - @view.extras[:auth_count], "other #{title.singularize}")
         + " on this " + ui_lookup(:tables => @table_name)
       end
+    elsif @display == "containers"
+      title = ui_lookup(:tables => "containers")
+      drop_breadcrumb(:name => record.name + " (All #{title})",
+                      :url  => "/container_group/show/#{record.id}?display=#{@display}")
+      @view, @pages = get_view(Container, :parent => record) # Get the records (into a view) and the paginator
+      @showtype = @display
+      if @view.extras[:total_count] &&
+         @view.extras[:auth_count] &&
+         @view.extras[:total_count] > @view.extras[:auth_count]
```

Comment:
Prefer removing these first two nil checks in favor to something like this using Hash#fetch with a 0 default if missing:
```ruby
      if @view.extras.fetch(:total_count, 0) > @view.extras.fetch(:auth_count, 0)
```